compiler/p/codegen/GenerateInstructions.cpp

```diff
@@ -683,11 +683,11 @@ void generateZeroExtendInstruction(TR::Node *node,
                                    int32_t bitsInTarget,
                                    TR::CodeGenerator *cg)
    {
-   TR_ASSERT((TR::Compiler->target.is64Bit() && bitsInTarget > 0 && bitsInTarget < 64) ||
-          (TR::Compiler->target.is32Bit() && bitsInTarget > 0 && bitsInTarget < 32),
+   TR_ASSERT((cg->comp()->target().is64Bit() && bitsInTarget > 0 && bitsInTarget < 64) ||
+          (cg->comp()->target().is32Bit() && bitsInTarget > 0 && bitsInTarget < 32),
           "invalid zero extension requested");
```

Comment:
Just verifying have you ran under a debug build to validate all the replacements inside of such macros are working correctly?

Reply:
I know the linter did catch a couple of errors within `TR_ASSERT`. However I haven't tried to build a debug build so I'll try to do that.

Reply:
Let me know once you have tested. Also there are some conflicts to resolve now.

Reply:
Debug builds pass no problem. 

>  Also there are some conflicts to resolve now.
😞